site/_posts/2019-10-01-implementing-fusionauth-python.md

```diff
@@ -355,7 +367,7 @@ Now, the moment we've been waiting for. Execute the test file we just created.
 (venv) $ python test.py
 ```
 
-If successful, our application should return a client response like this:
+If successful, your application should return a client response like this:
 
 ```json
```

Comment:
This looks like Python, not JSON because `True` isn't a valid JSON value (`true` is).

What is the output? Should this block use the python syntax highlighter or the JSON one?

Reply:
fixed in latest commit